### PR TITLE
[Module 3] components

### PR DESCRIPTION
**Pull request by VolhaChubakova**

### CODE REVIEW

File: module_1/src/stories/Button.jsx [hunk 1]
```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import './button.css';
+
+/**
+ * Primary UI component for user interaction
```
Comment (supercilium) [inline module_1/src/stories/Button.jsx]: Are these files related to your project? I suppose they are examples and you don't need to keep them in your code base

File: cypress/e2e/spec2.cy.js [hunk 2]
```diff
@@ -0,0 +1,5 @@
+describe('template spec', () => {
+  it('passes', () => {
+    cy.visit('https://example.cypress.io')
```
Comment (supercilium) [inline cypress/e2e/spec2.cy.js]: The same here - if these files just examples from the framework documentation - no need to keep them in your project

**Review by supercilium — APPROVED** [review]

File: cypress/e2e/spec2.cy.js (hunk 2 above)
Comment (VolhaChubakova) [reply cypress/e2e/spec2.cy.js]: Oh yes, these files are redundant. Thanks for the feedback!
